### PR TITLE
Kc readme

### PR DESCRIPTION
**Pull request by KeeshaCain**
The initial outline of the README, has a few gaps of things that I'm unsure of.

### CODE REVIEW

**Comment by ed42311** [comment]
@KeeshaCain looks like you got the hang of markdown 🛩 header tags need a space after the has to compile correctly ## thing not ##thing

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+* Laravel
+* Php
+* SASS
+* Vue.js
+* OAuth
+* Bootstrap
+### Backend
+* MySql
+* Eloquent
+###Testing gear
```
Comment (ed42311) [inline README.md]: a sperce needed

File: README.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+* Phpunit
+* Jasmine
+###Communication
```
Comment (ed42311) [inline README.md]: nutha sperc

File: README.md [hunk 3]
```diff
@@ -1 +1,97 @@
...
+
+## Standards
+---
+### Github
+
+* No more than 3 branches
+  * main working branch
+  * side project branch
+  * safety branch
+* Branches must have a name of what your are working on (example: n8-backend-server-setup)
```
Comment (ed42311) [inline README.md]: awesome, suggest categories  such as `n8-bug-something`

File: README.md [hunk 4]
```diff
@@ -1 +1,97 @@
...
+* **Nate**
+  * Project Manager
+
+## File Structure
+---
+```
+    Update this later
+```
+
+## Standards
```
Comment (ed42311) [inline README.md]: suggest airbnb standards doc

File: README.md [hunk 5]
```diff
@@ -1 +1,97 @@
...
+* Php
+* SASS
+* Vue.js
+* OAuth
+* Bootstrap
+### Backend
+* MySql
+* Eloquent
+###Testing gear
+* Phpunit
```
Comment (ed42311) [inline README.md]: do we need to add puppeteer for e2e testing @iverunamok?

File: README.md [hunk 6]
```diff
@@ -1 +1,97 @@
 # jeux
+---
+Game night.
+We all love it, we all do it, we all have a special "gamer tag" that defines who we are in that space.
+Ever wanted to try out a game to see if it's worth the price or the time? I bet you have. We have the solution for you!
+Jeux is a handy site that lets you and your friends share games with one another while allow you to keep track of who has what. No more wondering where your Friday night specialty has gone! Jeux has you covered!
+
+## Table of Contents
+---
+- [What's inside (Technologies)](#What's-inside-Technologies)
```
Comment (ed42311) [inline README.md]: to correctly navigate these will need anchor tags <a name="of-the-thing-to-navigate-to"> </a>  no spaces no errant characters  as in ' / etc

File: README.md (hunk 5 above)
Comment (iverunamok) [reply README.md]: Yes Please :D will we also list Headless Chrome in here? or would that be considered part of Puppeteer?

Comment (ed42311) [reply README.md]: good question, look it up :)

**Comment by freethewhat** [comment]
This pull request will resolve #8. 

Please make sure you are linking the issue the pull request will resolve.

**Pull request merged**